tests/gradients/test_custom_jacobian.py

```diff
@@ -0,0 +1,35 @@
+import pennylane.numpy as np
+import pennylane as qml
+
+from pennylane.devices.default_qubit import DefaultQubit
+
+class CustomJacobianDevice(DefaultQubit):
```

Comment:
This would best belong in the `tests/interfaces/autograd.py` file as it's an autograd interface specific test.

Reply:
@puzzleshark also: don't forget to add an entry to the changelog